README.md

```diff
@@ -160,13 +160,13 @@ digitalocean          Opaque                                1         18h
 Before you continue, be sure to checkout to a [tagged
 release](https://github.com/digitalocean/csi-digitalocean/releases). Always use the [latest version](https://github.com/digitalocean/csi-digitalocean/releases) compatible with your Kubernetes release (see the [compatibility information](#kubernetes-compatibility)).
 
-The [releases directory](deploy/kubernetes/releases) directory holds manifests for all plugin releases. You can deploy a specific version by executing the command
+The [releases directory](deploy/kubernetes/releases) holds manifests for all plugin releases. You can deploy a specific version by executing the command
 
 ```shell
-kubectl apply -f https://raw.githubusercontent.com/digitalocean/csi-digitalocean/master/deploy/kubernetes/releases/csi-digitalocean-vX.Y.Z.yaml
+kubectl apply -f https://raw.githubusercontent.com/digitalocean/csi-digitalocean/master/deploy/kubernetes/releases/csi-digitalocean-vX.Y.Z
```

Comment:
Does this work? I don't have a cluster that I can try this change on right now but I tried a dry-run on one cluster and got an error:

```
$ oc apply --dry-run=client -f https://raw.githubusercontent.com/digitalocean/csi-digitalocean/master/deploy/kubernetes/releases/csi-digitalocean-v2.0.0 
error: unable to read URL "https://raw.githubusercontent.com/digitalocean/csi-digitalocean/master/deploy/kubernetes/releases/csi-digitalocean-v2.0.0", server reported 404 Not Found, status code=404
```

Reply:
You're right, I missed that we cannot use the raw link with a directory.

Given that our documentation already states to check out a tag of the repo, I updated the docs to reference a directory. I also (manually) incorporated your suggestion above.